romana-install/config.yml

```diff
@@ -0,0 +1,36 @@
+---
+# Ensure stack nodes are ready for installation
```

Comment:
It seems to be an ubuntu thing. Both AWS images and Vagrant images have this cloud-init thing present, so I didn't make it specific to the platform type.